[3.7.0] docs(scheduler): OnCellRender event (#1181)
telerik/blazor-docs: components/scheduler/events.md

</style>
````

## OnCellRender

> The `OnCellRender` event fires upon rendering of each slot cell. It allows you set custom CSS class to the cells based on the business logic.

Reviewer: ```suggestion
The `OnCellRender` event fires upon rendering of each slot cell. It allows you to set a custom CSS class to the cell, based on business logic.
```

The handler receives an argument of type `SchedulerCellRenderEventArgs` which exposes the following fields:

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Field | Type | Description |
| --- | --- | --- |
| `Class` | `string` | The CSS class that will be applied to the cell. |

> | `Date` | `DateTime` | The actual date that is associated with the cell. |

Author: ```suggestion
| `Date` | `DateTime` | The date that is associated with the cell. |
```

> | `Class` | `string` | The CSS class that will be applied to the cells. The CSS rules that are set for that class will be visibly rendered on the Scheduler cells. |

Reviewer: The second sentence is self-evident, and at the same time, it guarantees something that is not that certain :)

```suggestion
| `Class` | `string` | The CSS class that will be applied to the cell. |
```

> | `Date` | `DateTime` | The actual date that is associated with the cell. |

Reviewer: What additional value does "actual" bring?

Author: It doesn't, I removed it.
